lib/register_sources_bods/register/statement_loader.rb

```diff
@@ -31,14 +31,16 @@ def load_statements(statement_ids, load_children: true)
       def load_statements_children(statements)
         new_statements = statements
 
-        level = 1
+        level = 0
         while !new_statements.empty? && (level <= MAX_LEVELS)
           new_statements = load_associated_statements(new_statements.keys, interested_party: false, subject: true).to_h { |r| [r.statementID, r] }.reject { |k, _v| statements.key? k }
 
           next_statement_ids = new_statements.keys
 
           next_statement_ids += statements.values.select { |s| s.respond_to?(:interestedParty) }.map(&:interestedParty).compact.map(&:describedByEntityStatement).compact
           next_statement_ids += statements.values.select { |s| s.respond_to?(:interestedParty) }.map(&:interestedParty).compact.map(&:describedByPersonStatement).compact
+          next_statement_ids += new_statements.values.select { |s| s.respond_to?(:interestedParty) }.map(&:interestedParty).compact.map(&:describedByEntityStatement).compact
```

Comment:
Consider extracting `new_statements.values.select { |s| s.respond_to?(:interestedParty) }.map(&:interestedParty).compact`. Similarly for `statements.values.select { |s| s.respond_to?(:interestedParty) }.map(&:interestedParty).compact`.